pkg/api/applications/v2/recommendation.go

```diff
@@ -122,6 +123,40 @@ func (rl *ResourceList) Set(name string, value api.NumberOrString) {
 	}
 }
 
+type HPAResources struct {
+	Bounds *HPABounds `json:"bounds,omitempty"`
+}
+
+type HPABounds struct {
+	TargetUtilization *HPABoundsRange `json:"targetUtilization,omitempty"`
+}
+
+type HPABoundsRange struct {
+	Max *HPAResourceList `json:"max,omitempty"`
+	Min *HPAResourceList `json:"min,omitempty"`
```

Comment:
I think we should just use the stock `ResourceList` here instead of adding a new type. That is more consistent with the Kube API.

Reply:
I'm fine over-specifying the domain to simplify the code, it would be nicer if we could just add `TargetUtilization *BoundsRange` to the existing `Bounds` type.

Reply:
We might want `0-100` validation somewhere in [here](https://github.com/thestormforge/optimize-go/blob/efd315f9fbf63ff6eb3865d50d34741e425b7a48/pkg/command/recommendation/options.go#L314), but again we have the problem with `len(configurations) > 1`.

Reply:
added integer and 0-100 validation.  added TargetUtilization to the existing Bounds type.